runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -328,7 +328,9 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		int itemId = event.getIdentifier();
 		String option = Text.removeTags(event.getOption()).toLowerCase();
 		String target = Text.removeTags(event.getTarget()).toLowerCase();
-
+		if (option.equals("climb") && target.contains("staircase"))
+			config.swapClimb();
+				swap("climb-up", option, target, true);
```

Comment:
config.swapClimb() needs to be inside the `if ()` check, otherwise the setting isn't actually being checked and the swap will always be performed.

Use separate-lined `{ }` braces to contain the if statement too, that's another one of our code conventions.